package/src/components/Channel/Channel.tsx

```diff
@@ -672,6 +674,22 @@ const ChannelWithContext = <
     }
   };
 
+  const handleAppBackground = useCallback(() => {
+    if (channel) {
+      channel.sendEvent({ parent_id: thread?.id, type: 'typing.stop' } as StreamEvent<
+        At,
+        Ch,
+        Co,
+        Ev,
+        Me,
+        Re,
+        Us
+      >);
+    }
+  }, [thread?.id, channelId]);
+
+  useAppStateListener(undefined, handleAppBackground);
+
```

Comment:
Move this to line 612 (before `* CHANNEL CONSTANTS`).

copyTypingState, copyReadState, copyChannelState etc all these functions are part of following handleEvent and useEffect. So let's not disrupt that order.